src/components/Staking/WithdrawalsTabComparison.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react"
+import {
+  Button,
+  Heading,
+  ListItem,
+  Tab,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Tabs,
+  Text,
+  UnorderedList,
+} from "@chakra-ui/react"
+import WithdrawalCredentials from "./WithdrawalCredentials"
+import ButtonLink from "../ButtonLink"
+
+interface IProps {}
+const WithdrawalsTabComparison: React.FC<IProps> = () => (
+  <Tabs>
+    <TabList>
+      <Tab>Current stakers</Tab>
+      <Tab>New stakers (not yet deposited)</Tab>
+    </TabList>
+
+    <TabPanels>
+      <TabPanel>
+        <Heading as="h3">Current stakers</Heading>
+        <UnorderedList>
+          <ListItem>
+            Some users may have provided a withdrawal address when initially
```

Comment:
```suggestion
            Certain users may have provided a withdrawal address during initial
```

Reply:
Any reason for this rewording?

(Also, friendly GitHub note... you can drag down when selecting lines in a review to do a multi-line comment/code suggestion, so these don't have to be split up =) [see this StackOverflow answer](https://stackoverflow.com/questions/48911087/how-to-select-comment-on-a-range-of-lines-in-github-pull-request#58190804))

Reply:
Oh TIL, great to know! Thanks.

The rewording was just to add a formal tone to the documentation. Totally up to you guys to decide.